packages/typespec-client-generator-core/README.md

```diff
@@ -508,25 +535,32 @@ op myOperation(): void;
 
 #### `@clientInitialization`
 
-Customize the client initialization way.
+Customize the client initialization way in the generated client SDK.
```

Comment:
```suggestion
Customize the client initialization way in the generated client SDK.
```

Allows customization of how clients are initialized in the generated SDK.
By default, the root client is initialized independently, while sub-clients are initialized through their parent client.
Initialization parameters typically include endpoint, credential, and api-version.